app/controllers/reviews_controller.rb

```diff
@@ -21,6 +24,34 @@ def create
     end
   end
 
+  def edit
+    @venue = Venue.find(params[:venue_id])
+    @review = Review.find(params[:id])
+    @quantity_collection = [[1, 1], [2, 2], [3, 3], [4, 4], [5, 5]]
+  end
+
+  def update
+    @quantity_collection = [[1, 1], [2, 2], [3, 3], [4, 4], [5, 5]]
+    updated_params = review_params
+    updated_params[:user_id] = current_user.id
```

Comment:
I honestly wouldn't do this, what if an admin edits this review? If an admin does this, then the review owner will become the admin